Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md

```diff
@@ -147,6 +183,10 @@ Any media files you already have on your site will not automatically be added to
 
 Any new media files you upload to the site, will automatically be added to the Blob Storage.
 
+## Exisiting Forms files
+
+Any Umbraco forms data files will not automatically be added to blob storage. You will need to upload the **contents** of the Umbraco forms data folder to storage. This can be found in `~/App_Plugins/UmbracoForms/Data`. Again you need the **contents** of the data folder so the **forms**, **templates** and **workflows** folders should be the only folders in the container. Do not upload the whole data folder! 
```

Comment:
```suggestion
Any existing Umbraco Forms data files will not automatically be added to the blob storage. You will need to upload the **contents** of the Umbraco Forms data folder to the storage. This can be found in `~/App_Plugins/UmbracoForms/Data`. Again you need the **contents** of the Data folder so the **forms**, **templates** and **workflows** folders should be the only folders in the container. Do not upload the whole Data folder. 
```

Reply:
"the blob storage" and "the storage" makes sense but sounds wrong. I guess changing it to be this would be better: "automatically be added to Azure Blob storage" "Umbraco Forms data folder to Blob storage". 

I am not sure how to change your suggestion. "Add suggestion to batch" is greyed out for me

Reply:
I see your point 🤔
I've tried to update the wording a bit..
Instead of saying "the Blob storage", I've left it out all together, so it now says: "You will need to upload the contents of the Umbraco Forms data folder manually."

I think that works better. What do you think?